src/System.Configuration/ref/System.Configuration.cs

```diff
@@ -1356,10 +1454,12 @@ public partial interface IConfigSystem
         System.Configuration.Internal.IInternalConfigRoot Root { get; }
         void Init(System.Type typeConfigHost, params object[] hostInitParams);
     }
+    [System.Runtime.InteropServices.ComVisibleAttribute(false)]
```

Comment:
Do we need these?  What are we doing for other assemblies?  If it isn't needed leave it off.